image-download

```diff
@@ -45,8 +45,8 @@ import urllib.parse
 
 from lib.constants import IMAGES_DIR
 from lib.directories import get_images_data_dir
+from lib.stores import PUBLIC_STORES, REDHAT_STORES, CA_PEM
```

Comment:
Let's drop the `As far as I know..." from the commit message. I checked `for d in *; do git -C $d grep STORES; done` in my ~/upstream/ dir (same for `CA_PEM`), and it's indeed only being used in bots itself.